packages/contracts/contracts/test/Colosseum.t.sol

```diff
@@ -55,17 +50,38 @@ contract MockColosseum is Colosseum {
     ) external view returns (bool) {
         return _isAbleToBisect(challenges[_outputIndex][_challenger]);
     }
+}
 
-    function hashPublicInput(
+contract MockZKProofVerifier is ZKProofVerifier {
+    constructor(
+        ZKVerifier _zkVerifier,
+        bytes32 _dummyHash,
+        uint256 _maxTxs,
+        address _zkMerkleTrie,
+        ISP1Verifier _sp1Verifier,
+        bytes32 _zkVMProgramVKey
+    )
+        ZKProofVerifier(
+            _zkVerifier,
+            _dummyHash,
+            _maxTxs,
+            _zkMerkleTrie,
+            _sp1Verifier,
+            _zkVMProgramVKey
+        )
+    {}
+
+    function hashZKEVMPublicInput(
         Types.PublicInputProof calldata _proof
     ) external view returns (bytes32) {
-        return _hashPublicInput(_proof.srcOutputRootProof.stateRoot, _proof.publicInput);
+        return _hashZKEVMPublicInput(_proof.srcOutputRootProof.stateRoot, _proof.publicInput);
```

Comment:
The location of the _hashZKEVMPublicInput function has changed, shouldn't this call be a problem?

Reply:
That's the reason why I added this new `MockZKProofVerifier` which inherits `ZKProofVerifier`.